src/StructuredLogger/StructuredLogger.cs

```diff
@@ -103,10 +105,14 @@ public override void Shutdown()
 
             if (projectImportsCollector != null)
             {
-                var bytes = projectImportsCollector.GetAllBytes();
-                construction.Build.SourceFilesArchive = bytes;
-
                 projectImportsCollector.Close();
+
+                

```

Comment:
I like to avoid two consecutive empty lines in .cs files, perhaps I should add an .editorconfig rule for this.

Reply:
I wanted to add a rule - but it seems it's not possible :-( https://github.com/editorconfig/editorconfig/issues/336#issuecomment-372373305